src/pages/LoginPage.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "../style/App.scss";
+import smilies from "../assets/smilies.png";
+import i6 from "../assets/i6.png";
+import social from "../assets/social.png";
+import play from "../assets/play.png";
+import appStoreButton from "../assets/appstore_button.png";
+import menuIcon from "../assets/icons/menu_icon.png";
+import LoginForm from "../components/users/LoginForm"
+
+function Login() {
+    return (
+      <div className="main-div">
+  
+          <div className="left-container">
+  
+            <div className="left-container__menu">
+              <img src={menuIcon} alt="menu_button"></img>
+            </div>
+  
+            <div className="left-container__smilies">
+              <img src={smilies} alt="smilies"></img>
+            </div>
+  
+            <p className="left-container__app-title">TARGET MVD</p>
+            <p className="left-container__app-subtitle">Find people near you & connect</p>
+            <p className="left-container__description-text" >Create a  target wherever on the map, specify your interest: 
+              Travel, Dating, Music, etc and start conecting with others who share your interest.</p>
```

Comment:
You should consider using `react-intl` to localize your web. Then you can import the right translation of the strings in each screen